src/ecdsa/__init__.py

```diff
@@ -19,6 +19,7 @@
     BRAINPOOLP320r1,
     BRAINPOOLP384r1,
     BRAINPOOLP512r1,
+    Curve25519Weier,
```

Comment:
I'd prefer the full name "Curve25519Weierstrass"
but then I've found at least https://tools.ietf.org/id/draft-struik-lwip-curve-representations-00.html referring to it as "Wei25519"

Reply:
> 1. What's the usefulness of a Weirstrass representation of Curve25519? It won't interoperate with X25519 or Ed25519 implementations without additional code (projecting the results back and forth between Weierstrass and Edwards representations)
>    
>    * To be clear, I have nothing against adding support for X25519 or Ed25519, but I don't think the high level interface should expose the Weirstrass representation used on the backend
> 2. Test coverage missing
> 3. Please fix the email used in the commit: either change it to the one you use for github or add it as an alternative one in your profile (if you are the author)

1. I have to communicate with Bouncy Castle. This library implemented Curve25519 as Weierstrass Form.
2. I could not run the test with tox tool, how could I deal with this problem? Should I open new issue?
3. I changed it.

Thank you for your feedback.

Reply:
That PR in Javascript library uses different name still... How does BouncyCastle call it?

Reply:
Just `Curve25519` :-)

Reply:
yeah, though the docs are explicit that it's  non-standard: https://github.com/bcgit/bc-java/blob/738dfc0132323d66ad27e7ec366666ed3e0638ab/core/src/main/javadoc/org/bouncycastle/math/ec/custom/djb/package.html

so I'd prefer the https://tools.ietf.org/html/draft-ietf-lwig-curve-representations-11 naming